src/pages/home/home--logged-in.stories.js

```diff
@@ -534,136 +529,152 @@ const Template = (args) => (
             text-size="s"
+            variant="block"
           >
-            SBB News
+            Complaints
           </sbb-link>
           <sbb-link
             href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
             text-size="s"
+            variant="block"
           >
-            SBB Community
+            Praise
           </sbb-link>
           <sbb-link
             href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
             text-size="s"
+            variant="block"
           >
-            Company
+            Report property damage
           </sbb-link>
         </sbb-link-list>
+        <sbb-button
+          variant="primary"
+          size="m"
+          label="All help topics"
+          event-id="Event ID"
+          name="sample-name"
+          value="sample-value"
+        ></sbb-button>
       </div>
-      <div slot="col-3">
-        <sbb-stack>
-          <sbb-title level="2" visual-level="5">
-            Newsletter.
-          </sbb-title>
-          <p>Our newsletter regularly informs you of attractive offers from SBB via e-mail.</p>
-          <sbb-stack space-leading="fixed-3x">
+      <sbb-link-list title-level="2" title-content="More SBB.">
+        <sbb-link
+          href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+          text-size="s"
+          variant="block"
+        >
+          Jobs & careers
+        </sbb-link>
+        <sbb-link
+          href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+          text-size="s"
+          variant="block"
+        >
+          Rail traffic information
+        </sbb-link>
+        <sbb-link
+          href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+          text-size="s"
+          variant="block"
+        >
+          SBB News
+        </sbb-link>
+        <sbb-link
+          href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+          text-size="s"
+          variant="block"
+        >
+          SBB Community
+        </sbb-link>
+        <sbb-link
+          href="https://www.sbb.ch/de/hilfe-und-kontakt/erstattung-entschaedigung/rueckerstattung-von-billetten.html"
+          text-size="s"
+          variant="block"
+        >
+          Company
+        </sbb-link>
+      </sbb-link-list>
+      <div className="sbb-link-list-button-group">
+        <sbb-link-list title-level="2" title-content="Newsletter.">
+          <sbb-link text-size="s" variant="block">
+            Our newsletter regularly informs you of attractive offers from SBB via e-mail.
+          </sbb-link>
+          <span slot="button">
             <sbb-link-button
```

Comment:
Please notice that the link-button will be removed in [the button PR](https://github.com/lyne-design-system/lyne-components/pull/1328).
Same for line 203 and 506.

Reply:
@DavideMininni-Fincons I know, but until the button is done, as far as I understood right, it should stay with the SBB-Link-Button.